### ISSUE TITLE
UI: update disaster recovery warning buttons in Settings

### ISSUE THREAD

**Issue by its-me-abhishek** [ZeusLN/zeus#2861]
### Describe your enhancement idea

Currently the buttons in the Disaster Recovery warning-buttons appear to have low contrast, making them hard to read. To fix this, we can use a brighter background (e.g., red with white text or yellow with black text - to fit the Zeus color scheme), or something similar to the  Backup wallet button, to improve readability.

1. Disaster Recovery
![Image](https://github.com/user-attachments/assets/7c71c58d-d9c1-453f-a9cc-cfd33d778071)

2. Backup Wallet
![Image](https://github.com/user-attachments/assets/793150c4-09f6-46ae-9b5a-447d0e6a3631)

**Comment by its-me-abhishek** [comment]
I would like to work on this issue if it is assignable

**Comment by kaloudis** [comment]
> I would like to work on this issue if it is assignable

Sure, it's all yours
